test/cpp/end2end/xds/xds_cluster_end2end_test.cc

```diff
@@ -1494,65 +1494,73 @@ TEST_P(ClientLoadReportingTest, SendAllClusters) {
   balancer_->ads_service()->SetEdsResource(BuildEdsResource(args));
   // Wait until all backends are ready.
   size_t num_warmup_rpcs = WaitForAllBackends(DEBUG_LOCATION);
-  // Send kNumRpcsPerAddress RPCs per server.
-  xds::data::orca::v3::OrcaLoadReport backend_metrics;
-  auto& named_metrics = (*backend_metrics.mutable_named_metrics());
-  named_metrics["foo"] = 1.0;
-  named_metrics["bar"] = 2.0;
-  CheckRpcSendOk(DEBUG_LOCATION, kNumRpcsPerAddress * backends_.size(),
-                 RpcOptions().set_backend_metrics(backend_metrics));
-  named_metrics["foo"] = 0.3;
-  named_metrics["bar"] = 0.4;
-  for (size_t i = 0; i < kNumFailuresPerAddress * backends_.size(); ++i) {
-    CheckRpcSendFailure(DEBUG_LOCATION, StatusCode::FAILED_PRECONDITION, "",
-                        RpcOptions().set_server_fail(true).set_backend_metrics(
-                            backend_metrics));
-  }
-  // Check that each backend got the right number of requests.
-  for (size_t i = 0; i < backends_.size(); ++i) {
-    EXPECT_EQ(kNumRpcsPerAddress + kNumFailuresPerAddress,
-              backends_[i]->backend_service()->request_count());
+  for (int report = 0; report < 3; ++report) {
```

Comment:
This change should be made in the `Vanilla` test above, not here in the `SendAllClusters` test.

Also, please add a comment explaining why we are doing this several times.

Reply:
Updated.